game/scripts/vscripts/libraries/garbagecollection.lua

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
There are a lot of files that do this though...
I'll add some comments about usage.